data/myapp.desktop

```
@@ -0,0 +1,18 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+
+Name=MyApp
+Comment=A catchy tagline
+Categories=Education;ComputerScience;
+
+Icon=io.github.myteam.myapp
+Exec=io.github.myteam.myapp
+Terminal=false
+
+Actions=quit;
+DBusActivatable=true
+
+[Desktop Action quit]
+Name=Quit
```

Would it make sense to showcase the action icon (it's mentioned in the Lauchers section of the docs). We currently show it in the right-click menu in the dock.

```suggestion
Name=Quit
Icon=process-stop
```